generated/proto/schema/schema.proto

```diff
@@ -53,14 +53,14 @@ message Database {
 	int64 write_cutover_time = 6;
 	int64 cutover_complete_time = 7;
 	map<string, Cluster> clusters = 8;
-	map<string, ClusterShardAssignment> shard_assignments = 9;
+	map<string, ShardSet> shard_assignments = 9;
 	int32 version = 10;
 	repeated ClusterMappingRuleSet mapping_rules = 11;
 }
 
-// ClusterShardAssignment captures the shards currently assigned to a cluster
-message ClusterShardAssignment {
-	repeated uint32 shards = 1;
+// ShardSet is a bitset of shards
+message ShardSet {
+	repeated uint64 bits = 1;
```

Comment:
Is using a bitset more ideal for over the wire config updates or mem usage or otherwise?